week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,17 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  var conversionUSD = price * 1.4;
+  return addComission(conversionUSD);
+}
+function convertToBRL(price) {
+  var conversionBRL = price * 4.9;
+  return addComission(conversionBRL);
+}
+function addComission(conversion) {
+  return conversion + conversion.toFixed(2) * 0.01; // adding 1% comission to converted price
```

Comment:
Good, this works.  But there is something to be careful of here.  .toString() has actually returned a *string* here, and it is being converted back to a number for you.  
It is much better to keep things as numbers as long as possible, and call toFixed() on the final number!

Reply:
will do that